sdk/keyvault/Azure.Security.KeyVault.Keys/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+## 4.1.0-preview.1
```

Comment:
I thought the docs stated these have to be L1 headings, implying `# Release History` should be removed from all changelogs and everything bumped up a level, or did/can we change that requirement? Changing the requirement (and any implementation thereof) would be a far smaller change, and I do prefer seeing "Release History" as a page title for those not paying attention to the file name / URI.

Reply:
You are right. I removed release history and made all version L1 so it consistent across all repos.

Reply:
@chidozieononiwu is there a reason that doc-warden isn't enforcing H1 for the versions?

Reply:
FWIW I agree with @heaths - I feel like the title brings value (and obviously other folks do too as many of these changelogs seem to have had a title which you've removed).

Reply:
Am jut following the existing change-log guidance which is also used to pull out release notes for github releases.

Reply:
I'm asking if it's possible to change the guidance, because I think having the H1 `# Release history` or `# Change history` adds clarity.

Reply:
Ok, I just discussed with Wes, I will adjust Doc Warden to accommodate other headers.

Reply:
@matthchr I have added the headers and updated the Doc Warden tool to accommodate it.